app/src/main/java/com/android/halanvendordiscovery/remote/RemoteClient.kt

```diff
@@ -0,0 +1,60 @@
+package com.android.halanvendordiscovery.remote
+
+import com.android.halanvendordiscovery.BuildConfig
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object RemoteClient {
+    @Provides
+    @Singleton
+    fun provideGsonBuilder(): Gson {
+        return GsonBuilder()
+            .setLenient()
+            .create()
+    }
+
+    @Provides
+    @Singleton
+    fun provideLoggingInterceptor(): HttpLoggingInterceptor {
+        return HttpLoggingInterceptor()
```

Comment:
change the code to only enable logging for debug builds

Reply:
Modified